main.go

```diff
@@ -18,6 +17,11 @@ import (
 	"github.com/google/go-containerregistry/pkg/v1/tarball"
 
 	cobra "github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kvcorev1 "kubevirt.io/api/core/v1"
+	v1beta1 "kubevirt.io/api/export/v1beta1"
+	kubecli "kubevirt.io/client-go/kubecli"
 	tar "kubevirt.io/containerdisks/pkg/build"
```

Comment:
Are all these dependencies required to create a single RPC call?

Reply:
Yes, unfortunately. This was the mostly time-consuming issue of this PR, writing the code took a couple of minutes.

Probably it would be smarter to have this PR in kubevirt-tekton-tasks, as it already contains all required dependencies.